### PR TITLE
Signup page

### PR DESCRIPTION
**Pull request by vishalg8454**
Added Sign-Up page. The page has following components:

- Navbar with Login, Favorite and Cart button.
- SignUp form with email and password input and
- Footer.

image

### CODE REVIEW

File: pages/signup-page/signup-page.css [hunk 1]
```diff
@@ -8,3 +8,38 @@
...
+    padding: 0.5rem;
+    caret-color: var(--color-primary);
+    border: 2px solid var(--color-primary);
+    font-size: 1rem;
+    width: 100%;
+}
+.btn-signup{
+    width: 100%;
+    margin: 1rem 0rem 0rem 0rem;
+    border-radius: 0rem;
```
Comment (agrawalshanu708) [inline pages/signup-page/signup-page.css:44]: use proper tag like margin-top & remove unnecessary code line 44

**Review by agrawalshanu708 — APPROVED** [review]
good to merge

File: pages/signup-page/signup-page.css (hunk 1 above)
Comment (vishalg8454) [reply pages/signup-page/signup-page.css:44]: Line 44 was added to remove the side one side effect from other CSS, will take care in future.

**Review by prachi-sahani — COMMENTED** [review]
code looks clean and is good to merge! 👍🏻

**Pull request merged**